docs/modules/ROOT/pages/index.adoc

```diff
@@ -631,8 +755,14 @@ A Trigger defines actions to take when monitored conditions are met. Triggers ca
 {
   "host": "smtp.gmail.com",
   "port": 465,
-  "username": "sender@example.com",
-  "password": "smtp_password",
+  "username": {
+    "type": "Plain",
```

Comment:
Why upper case Plain? 
Thinking there would be a log of wrong configs attempts because of that.

Maybe to support both versions.

We can make changes based on feedback....

Just wanted to double check was this intended.

Reply:
Good point, I think I can create some deserializer macro to allow case insensitive enums